.github/reviewers.yml

```diff
@@ -0,0 +1,17 @@
+# documentation of options: https://github.com/marketplace/actions/auto-request-review
+
+# eventually move this to using GitHub Teams (we have a developers team)
+# requires the use of a personal access token with access to teams
+# for now, leaving it as an explicit list
+reviewers:
```

Comment:
Is this the list of people who would get auto-tagged for reviews? I agree that we don't need to spam people about reviews yet. People can explicitly request a relevant review when they open their PR.